docs/index.md

```diff
@@ -116,26 +116,26 @@ title: Whiting index
 
 |     | A | B | C | D | E | F | G | H |
 |-----|---|---|---|---|---|---|---|---|
-|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | ? [t][t]<br>[![](w/page219a.gif)][P219] |
+|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | [N1](#N1), [t][t]<br>[![](w/page219a.gif)][P219] |
```

Comment:
The note N5 appears nowhere in the table. Perhaps like this:

```suggestion
|  1  | [diagr][A1], [N5]:[c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [N5]:[t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [N5]:[t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [N5]:[t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [N5]:[t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [N5]:[t][t] [![](w/page198a.gif)][P198] | [N1](#N1), [N5]:[t][t]<br>[![](w/page219a.gif)][P219] |
```

See also the other comment.